src/code42cli/config.py

```diff
@@ -98,7 +110,16 @@ def _set_username(self, new_value, profile):
         profile[self.USERNAME_KEY] = new_value.strip()
 
     def _set_ignore_ssl_errors(self, new_value, profile):
-        profile[self.IGNORE_SSL_ERRORS_KEY] = str(new_value)
+        if profile[self.IGNORE_SSL_ERRORS_KEY] == str(new_value):
+            profile[self.IGNORE_SSL_ERRORS_KEY] = str(False)
+        else:
+            profile[self.IGNORE_SSL_ERRORS_KEY] = str(new_value)
+
+    def _set_use_v2_file_events(self, new_value, profile):
+        if profile[self.USE_V2_FILE_EVENTS_KEY] == str(new_value):
```

Comment:
We should use `.get()` to try to read the key here (and other places where we access the value), because on upgrading, existing profiles won't have this key and we then throw a KeyError. 

Also, I'm not sure I like the "flip whatever the current value is" logic for these when the flag is provided. I think I'd expect running `profile create -n new --use-v2-file-events ...` and `profile update -n new --use-v2-file-events --password <updated>` to both result in the profile using v2 events.

We should probably just move from an `is_flag=True` option to an explicit `type=click.types.BOOL`, so the user would do `--use-v2-file-events=true/false` to set it explicitly, and default to None so if the option isn't passed we leave it unchanged.

I'm ok with changing the ssl option here to that too, even though it's technically a breaking change, that option probably gets used so little it's not worth worrying about.

Reply:
Okay, cool, I'd definitely prefer to move away from the flag so that sounds good to me. 👍